plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMapping.java

```diff
@@ -111,6 +112,40 @@ public void testNonLowerCaseTableName()
         }
     }
 
+    @Test
+    public void testNonLowerCaseViewName()
```

Comment:
As a follow-up maybe we should create an issue to test case-insensitive matching for views in all other connectors too - there is a base test class where we can add that.

Reply:
https://github.com/trinodb/trino/issues/9165